lib/internal/util/comparisons.js

```diff
@@ -5,6 +5,7 @@ const { isArrayBufferView } = require('internal/util/types');
 const { internalBinding } = require('internal/bootstrap/loaders');
 const { isDate, isMap, isRegExp, isSet } = internalBinding('types');
 const { getOwnNonIndexProperties } = process.binding('util');
+const { kPropertyFilter: { ONLY_ENUMERABLE } } = require('internal/util');
```

Comment:
Nit: while I use this style of destructuring in my own code - a common criticism I've found is that it's more confusing and in this case it might make sense to import the whole filter similarly to how V8 does `PropertyFilter:: ALL_PROPERTIES` although scoping rules would have allowed it to use the unqualified name allow.